packages/flutter_tools/templates/plugin_shared/pubspec.yaml.tmpl

```diff
@@ -6,7 +6,7 @@ homepage:
 environment:
   sdk: {{dartSdkVersionBounds}}
 {{#withPlatformChannelPluginHook}}
-  flutter: ">=2.5.0"
+  flutter: ">=3.3.0"
 {{/withPlatformChannelPluginHook}}
 {{#withFfiPluginHook}}
   flutter: ">={{minFrameworkVersionFfiPlugin}}"
```

Comment:
This is 2.11, so the branching for `flutter:` here, and `minFrameworkVersionFfiPlugin` in all of the code, can be eliminated now. It only existed because the platform channel version was lower than the FFI version, which is no longer the case.